test/fixtures/output/network/dns/forwarder/describe.golden

```diff
@@ -0,0 +1,11 @@
++----------------+---------------------------------+
+| ID             | dnsf-abcde1                     |
+| Name           | my-dns-forwarder                |
+| Domains        | abc.com, def.com,               |
+|                | example.domain, xyz.com,        |
+|                | my.dns.forwarder.example.domain |
```

Comment:
Would it be helpful to sort this list?

Reply:
sorting alphabetically would be useful for sure